src/sentry/dynamic_sampling/__init__.py

```diff
@@ -0,0 +1,32 @@
+from typing import List
+
+import sentry_sdk
+
+from sentry import quotas
+from sentry.dynamic_sampling.feature_multiplexer import DynamicSamplingFeatureMultiplexer
+from sentry.dynamic_sampling.utils import BaseRule, generate_environment_rule, generate_uniform_rule
+from sentry.models import Project
+
+
+def generate_rules(project: Project) -> List[BaseRule]:
+    """
+    This function handles generate rules logic or fallback empty list of rules
+    """
+    rules = []
+
+    sample_rate = quotas.get_blended_sample_rate(project)
+
+    boost_environments = DynamicSamplingFeatureMultiplexer.get_user_bias(
+        "boostEnvironments", project.get_option("sentry:dynamic_sampling_biases", None)
+    )
```

Comment:
Lets move this after the sample rate check

Reply:
fixed.